Phenotypes.pm

```diff
@@ -116,16 +123,15 @@ sub new {
     my $version = $config->{db_version} || 'Bio::EnsEMBL::Registry'->software_version;
     my $assembly = $config->{assembly};
 
-    my $dir = $self->params->[0];
+    my $dir = $DEFAULTS{dir};
     if(defined $dir && -d $dir){
       if( $species eq 'homo_sapiens' || $species eq 'human'){
-        my $assembly = $config->{human_assembly};
+        $assembly ||= $config->{human_assembly};
         $DEFAULTS{file} = sprintf("%s_%s_%i_%s.bed.gz", $dir.$pkg, $species, $version, $assembly);
```

Comment:
Thanks @ima23 - this update will be useful.

A small thing - when creating the file using the input directory, can you ensure there is a '/' after the directory string? Otherwise a file called /path/directoryPhenotypes.pm_homo_sapiens_94_GRCh38.bed.gz is created.

Reply:
True, fixed in next commit.